CallAutomation_CallLiveTranscription/readme.md

```diff
@@ -23,43 +21,32 @@ It accepts an incoming call from a phone number, performs DTMF recognition, and
 2. git clone `https://github.com/Azure-Samples/Communication-Services-dotnet-quickstarts.git`.
 3. Navigate to `CallAutomation_CallLiveTanscription` folder and open `CallAutomation_CallLiveTanscription.sln` file.
 
-### Setup and host ngrok
-
-You can run multiple tunnels on ngrok by changing ngrok.yml file as follows:
-
-1. Open the ngrok.yml file from a powershell using the command ngrok config edit
-2. Update the ngrok.yml file as follows:
-    authtoken: xxxxxxxxxxxxxxxxxxxxxxxxxx
-    version: "2"
-    region: us
-    tunnels:
-    first:
-        addr: 8080
-        proto: http 
-        host_header: localhost:8080
-    second:
-        proto: http
-        addr: 5001
-        host_header: localhost:5001
-NOTE: Make sure the "addr:" field has only the port number, not the localhost url.
-3. Start all ngrok tunnels configured using the following command on a powershell - ngrok start --all
-4. Once you have setup the websocket server, note down the the ngrok url on your server's port as the websocket url in this application for incoming call scenario. Just replace the https:// with wss:// and update in the appsettings.json file.
+### Setup and host your Azure DevTunnel
+
+[Azure DevTunnels](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/overview) is an Azure service that enables you to share local web services hosted on the internet. Use the commands below to connect your local development environment to the public internet. This creates a tunnel with a persistent endpoint URL and which allows anonymous access. We will then use this endpoint to notify your application of calling events from the ACS Call Automation service.
+
+```bash
+devtunnel create --allow-anonymous
+devtunnel port create -p 5165
```

Comment:
please make sure your port here and launch settings port are same? Looks like launch settings is set to port 8080, please update here

Reply:
I have updated.